Combobox.js

```diff
@@ -767,6 +875,22 @@ define([
 			};
 		}),
 
+		// HasDropDown#_dropDownKeyUpHandler() override.
+		// Do not call openDropDown if widget does not have a down arrow shown.
+		_dropDownKeyUpHandler: dcl.superCall(function (sup) {
+			return function () {
+				if (this.hasDownArrow) {
+					if (this.inputNode.value.length === this.minFilterChars === 0) {
+						this.list.query = this.getInitialQuery;
+					} else if (this.inputNode.value.length < this.minFilterChars) {
+						return;
+					}
+
+					sup.call(this);
+				}
+			};
+		}),
```

Comment:
Sure. I'll do. Basically, if the widget does not have an down arrow, this means the widget is configured in auto-complete mode. In this mode we cannot open the dropdown; it will be opened once the user typed sometime, with `something.length > this.minFilterChars`.